modules/ROOT/pages/_partials/api-mgr.adoc

```diff
@@ -323,14 +305,21 @@ Besides the default `--help` flag, this command also accepts:
 
 Manages a new API, API version, or new API instance with the Exchange asset passed in `<assetId>`, and the version passed in `<assetVersion>`.
 
-Besides the default `--help` flag, this command also accepts:
+If you do not indicated if this is a Flex Gateway or Mule API instance, the command assumes it is a Flex Gateway instance.
```

Comment:
The command of manage will assume it is a deploy type of Hybrid in case of a Flex flag, but will not assume for default it is a Flex Gateway